### ISSUE TITLE
bmcbutler does not build with go get

### ISSUE THREAD

**Issue by gebi** [bmc-toolbox/bmcbutler#1]
Awesome tool, though does not build for me.

```
% go get -u github.com/bmc-toolbox/bmcbutler
...
# github.com/bmc-toolbox/bmcbutler/butler
go/src/github.com/bmc-toolbox/bmcbutler/butler/butler.go:86:50: cannot use "github.com/bmc-toolbox/bmcbutler/vendor/github.com/sirupsen/logrus".TextFormatter literal (type *"github.com/bmc-toolbox/bmcbutler/vendor/github.com/sirupsen/logrus".TextFormatter) as type "github.com/sirupsen/logrus".Formatter in argument to logging.SetFormatter:
        *"github.com/bmc-toolbox/bmcbutler/vendor/github.com/sirupsen/logrus".TextFormatter does not implement "github.com/sirupsen/logrus".Formatter (wrong type for Format method)
                have Format(*"github.com/bmc-toolbox/bmcbutler/vendor/github.com/sirupsen/logrus".Entry) ([]byte, error)
                want Format(*"github.com/sirupsen/logrus".Entry) ([]byte, error)
go/src/github.com/bmc-toolbox/bmcbutler/butler/butler.go:88:24: cannot use "github.com/bmc-toolbox/bmcbutler/vendor/github.com/sirupsen/logrus".DebugLevel (type "github.com/bmc-toolbox/bmcbutler/vendor/github.com/sirupsen/logrus".Level) as type "github.com/sirupsen/logrus".Level in argument to logging.SetLevel
```

additional infos: `go version go1.10.3 linux/amd64`

**Comment by joelrebel** [comment]
This should is fixed in 94d3da60, 6219a09a, will tag the first release.
Thanks for reporting!

**Comment by gebi** [comment]
many thx :)!
